ambry-api/src/main/java/com.github.ambry/config/FrontendConfig.java

```diff
@@ -57,15 +64,27 @@
   @Default("8192")
   public final Integer frontendChunkedGetResponseThresholdInBytes;
 
+  /**
+   * Boolean indicator to specify if unrecognized account and container should be prohibited for old put requests.
+   */
+  @Config("frontend.should.prohibit.old.put.with.unrecognized.account.and.container")
```

Comment:
Hard to give a nice name, have changed to `frontend.should.allow.unrecognized.account.and.container.for.pre.containerization.request`